src/Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.cpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "TemplatedLocalFunctions.hpp"
```

Comment:
Can you use the full path? We want to avoid possible collisions in the future. E.g.
```cpp
#include "Evolution/Particles/MonteCarlo/TemplatedLocalFunctions.hpp"
```